CONTRIBUTING.md

```diff
@@ -6,6 +6,7 @@ For pull requests, please stick to the following guidelines:
 
 * Add tests for any new features and bug fixes. Ideally, each PR should increase the test coverage.
 * Follow the existing code style. Run `make format` and `make lint` before checking in your code.
+  * This might involve previous runs of `make venv` and `make install`, in order to have proper local setup.
```

Comment:
Sorry you encountered some difficulties while setting LocalStack up.
Maybe we could rewrite that line with a link to https://docs.localstack.cloud/contributing/development-environment-setup/ to help new contributors?

Reply:
7e0411eb

Reply:
If you prefer something else, would you be so kind as to use the "suggestion" mechanism from GitHub? Thank you.

Reply:
Thanks a lot for the change, it's true, I could have used the "suggestion" mechanism first, sorry!